saleor/graphql/product/types/products.py

```diff
@@ -1142,3 +1186,33 @@ def resolve_url(root: models.ProductImage, info, *, size=None):
     @staticmethod
     def __resolve_reference(root, _info, **_kwargs):
         return graphene.Node.get_node_from_global_id(_info, root.id)
+
+
+@key(fields="id")
+class ProductImage(graphene.ObjectType):
+    id = graphene.ID(required=True, description="The ID of the image.")
+    alt = graphene.String(required=False, description="The alt text of the image.")
+    sort_order = graphene.Int(
+        required=False,
+        description=(
+            "The new relative sorting position of the item (from -inf to +inf). "
+            "1 moves the item one position forward, -1 moves the item one position "
+            "backward, 0 leaves the item unchanged."
+        ),
+    )
+    url = graphene.String(
+        required=True,
+        description="The URL of the image.",
+        size=graphene.Int(description="Size of the image."),
+    )
+
+    class Meta:
+        description = "Represents a product image."
+
```

Comment:
We should add custom id resolver.
```suggestion
    @staticmethod
    def resolve_id(root: models.ProductMedia, info):
        return graphene.Node.to_global_id("ProductMedia", root.id)
```

Currently, id si returned as database pk. 
![image](https://user-images.githubusercontent.com/43955230/108059217-760c9800-7055-11eb-90ff-0def81b927a0.png)